python/lsst/fgcmcal/utilities.py

```diff
@@ -774,3 +775,39 @@ def makeStdCat(stdSchema, stdStruct):
     stdCat['npsfcand'][:, :] = stdStruct['NPSFCAND'][:, :]
 
     return stdCat
+
+
+def computeApertureRadius(schema, fluxField):
+    """
+    Compute the radius associated with a CircularApertureFlux field or
+    associated slot.
+
+    Parameters
+    ----------
+    schema: `lsst.afw.table.schema`
+    fluxField: `str`
+       CircularApertureFlux field or associated slot.
+
+    Returns
+    -------
+    apertureRadius: `float`
+       Radius of the aperture field, in pixels.
+
+    Raises
+    ------
+    RuntimeError: Raised if flux field is not a CircularApertureFlux
+       or associated slot.
+    LookupError: Raised if fluxField is not found in schema.
```

Comment:
I'm not sure what the what the guidelines are on documenting a raise that is not explicitly from this function...it's coming from the call to 
`fluxFieldName = schema[fluxField].asField().getName()`
right?  Also, are you guaranteed that the only raise will be a LookupError from that call?

Reply:
It's the only one that I've found, but I'm removing it from the Raises.